rosdep/python.yaml

```diff
@@ -6472,6 +6472,17 @@ python3-defusedxml:
   opensuse: [python3-defusedxml]
   rhel: ['python%{python3_pkgversion}-defusedxml']
   ubuntu: [python3-defusedxml]
+python3-deprecated:
+  debian:
+    '*': [python3-deprecated]
+    buster: null
+  fedora: [python3-deprecated]
+  rhel: 
```

Comment:
Looks like there is trailing whitespace on this line that is causing CI to fail.